### ISSUE TITLE
how to setup node-lifx

### ISSUE THREAD

**Issue by batkhoai** [jdomeij/node-red-contrib-node-lifx#11]
I got bulb's IP and MAC, but do not know how to connect these to node-lifx (input and output)

**Comment by strelniece** [comment]
Had the same problem. Try to add server, then deploy and then add lamp.

**Comment by jdomeij** [comment]
Yes you need to deploy one server + light for the server to be initialized (it doesn't matter what light id you use), I have no clear solution of how to allow searching for lights the first time before deploying the server.
This problem only happens the first time when activating the lights so I have not put any effort into it

**Comment by hollyhudson** [comment]
Can this be put in the readme here and in the info for the module on the Node-RED page?  I banged my head against this forever before figuring out I had to hit "deploy" to activate the server, since I didn't have to do this for my Hue lights and their server/bridge/thing.  It would help newbies who don't automatically think to do this.  And thank you for the module!!!

**Comment by magnusboman** [comment]
What is this server suppose to be? It doesn't find any lights if leaving it empty (it says it's optional).

**Comment by batkhoai** [comment]
what should i do with these fields in Lifx Server?
Address
Broadcast
Lights

when click on search Light ID, I always get "No lights found"

**Comment by CircuitGuy** [comment]
@batkhoai Just to help future people (this has high google visibility).

I figured it out. Just put in your server details and "deploy" the diagram. It'll warn you that it's incomplete. Continue anyway. After it's deployed, you can edit the Lifx nodes again and the Light IDs will be visible.

**Comment by ericcastro** [comment]
me too I had no idea how to set this thing up until I read @hollyhudson comment, which explained a "Deploy" is first needed for a "lifx server" to launch.
I though I had to install some separate component and run a "lifx server" locally instead of it being taken care by itself.
I too agree, I should be on the README

**Comment by ristomatti** [comment]
Pull requests welcome at https://github.com/ristomatti/node-red-contrib-node-lifx.
